packages/utils/package.json

```diff
@@ -27,7 +27,7 @@
     "cross-env": "^7.0.3"
   },
   "dependencies": {
-    "date-fns": "^1.27.2"
+    "date-fns": "^2.22.1"

```

Comment:
The date patterns in date-fns 2.x is updated.
So it introduce a breaking change, at least for [formatToTimeZone](https://github.com/Talend/ui/blob/lmaillet/feat/date-formatting/packages/utils/src/date/index.ts#L157)

Reply:
great addition we always need to start upgrade somewhere to see the impact.
No impact on CDN so 👍